tests/profile/factoryboy.py

```diff
@@ -0,0 +1,7 @@
+import factory
```

Comment:
I like the addition of factoryboy but this is not used in this PR, right?
could you please remove it? 
thanks.